expr.go

```
@@ -2297,6 +2318,15 @@ func sneq(a, b Value, ctx *evalContext) (Value, error) {
 	return Bool(!val.Bool()), nil
 }
 
+func regexMatch(a, b Value) (Value, error) {
+	fmt.Println(a, b)
+	val, err := regexp.MatchString(b.String(), a.String())
+	if err != nil {
+		return Bool(false), nil
```

Should we return the error here or ignore it?